spec/priorityQueue.js

```diff
@@ -0,0 +1,126 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import priorityQueue from '../src/priorityQueue'
+
+chai.use(chaiChange)
+
+describe('priorityQueue', () => {
+  'use strict'
+
+  it('exists', () => {
+    expect(priorityQueue).to.be.a('function')
+  })
+
+  describe('enqueue()', () => {
+    it('pushes an element to the queue.', () => {
+      const queue = new priorityQueue()
+
+      expect(() => queue.enqueue('foo', '1'))
```

Comment:
Do you mean to pass the string `'1'`as the second parameter or the `Number` `1`?